src/main/resources/application.properties

```diff
@@ -14,6 +15,8 @@ logging.uris.exclude.regex=/health|/info|/ping|/swagger.*|/webjars/.*|/v2/api-do
 azure.application-insights.web.enable-W3C=true
 azure.application-insights.web.enable-W3C-backcompat-mode=true
 
+
+oauth.root.url = http://localhost:9090/auth
 whereabouts.api.client.id=whereabouts-api-client
 whereabouts.api.client.secret=clientsecret
-nomis.auth.url=http://localhost:9090/auth/oauth/token
+ouath.token.url=${oauth.root.url}/oauth/token
```

Comment:
these shouldn't be in both properties and yaml files, shouldn't need properties file at all...